kyaml/setters2/set_test.go

```diff
@@ -1343,7 +1343,7 @@ openAPI:
 			// invoke the setter
 			instance := &SetOpenAPI{
 				Name: test.setter, Value: test.value, ListValues: test.values,
-				SetBy: test.setBy, Description: test.description}
+				SetBy: test.setBy, Description: test.description, IsSet: true}
```

Comment:
Should we have a test that verifies the behavior when `IsSet` is `false`?

Reply:
@mortent Done.